### PR TITLE
Switch to coordinates objects, add self-identification property, game objects, better versioning.

### PR DESCRIPTION
**Pull request by bvanvugt**

### CODE REVIEW

File: apis/snake/spec.yaml [hunk 1]
```diff
@@ -99,10 +103,10 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Snake"
-    Game:
```
Comment (daniel-swu) [inline apis/snake/spec.yaml]: I find this name a bit odd, why the change from Game to SnakeRequest?

Comment (dlsteuer) [inline apis/snake/spec.yaml]: i find this name change a bit off, why the change from Game to SnakeRequest?
